projects/admin/src/app/migration/deduplication/record/search/migration-search/migration-search.component.html

```diff
@@ -0,0 +1,327 @@
+<!--
+  RERO angular core
+  Copyright (C) 2019-2024 RERO
+  Copyright (C) 2019-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<!-- primeng toaster -->
+<p-toast />
```

Comment:
Is it necessary here? Shouldn't it be in the main application?

Reply:
Yes but it is not the case for the moment. I will remove it on the next version.